docs/technical/preservation/file_repair.md

```diff
@@ -0,0 +1,72 @@
+# How to Demo File Repair
+
+## Create resources
+In the figgy staging instance, create 3 scanned resources which each have one image. Name them clearly so you can tell which is which, e.g.
+  * Test Repair Bad Cloud File
+  * Test Repair Bad Local File
+  * Test Repair Bad Cloud and Local Files
+
+Complete each resource so they all preserve to the cloud.
+
+## Corrupt a local file
+You'll want to corrupt the local file for both the "bad local file" and the "bad
+cloud and local files" objects.
+
+`$ bundle exec cap staging rails:console`
+
+```ruby
+# set an id variable to your fileset's id
+qs = ChangeSetPersister.default.query_service
+fs = qs.find_by(id: id)
+fs.file_metadata.first.file_identifiers.first
```

Comment:
Since we're in the console anyways can we corrupt the file from here? Something like
```ruby
file = Valkyrie::StorageAdapter.find_by(id: fs.file_metadata.first.file_identifiers.first)
File.open(file.disk_path, 'a') do |f|
  f.puts "p0wn3d"
end
```

?